spec/services/exports/case_log_export_service_spec.rb

```diff
@@ -63,6 +70,30 @@ def replace_entity_ids(export_template)
         export_service.export_case_logs
         expect(actual_content).to eq(expected_content)
       end
+
+      it "creates a logs export record in a database with correct time" do
+        export_service.export_case_logs
+        records_from_db = ActiveRecord::Base.connection.execute("select started_at, id from logs_exports ").to_a
+        expect(records_from_db[0]["started_at"]).to eq(time_now)
+        expect(records_from_db.count).to eq(1)
+      end
+
+      it "gets the logs for correct timeframe" do
+        start_time = Time.zone.local(2022, 4, 13, 2, 2, 2)
+        export = LogsExport.new(started_at: start_time, daily_run_number: 1)
+        export.save!
+        params = { from: start_time, to: time_now }
+        allow(CaseLog).to receive(:where).with("updated_at >= :from and updated_at <= :to", params).and_return([])
+        export_service.export_case_logs
```

Comment:
Is this expecting anything? Does it ever fail?

Reply:
Right, so the `allow` is an expect in this case, rubicon prefers that to `expect`. It does fail if we change the parameters in `with`

Reply:
Did you run rubocop without `bundle exec` before it?